tests/functional/filter.t

```diff
@@ -22,10 +22,22 @@ Filter with subsampling, requesting 1 sequence per group (for a group with 4 dis
   >  --metadata filter/metadata.tsv \
   >  --group-by region \
   >  --sequences-per-group 1 \
+  >  --subsample-seed 314159 \
   >  --output-strains "$TMP/filtered_strains.txt" > /dev/null
   $ wc -l "$TMP/filtered_strains.txt"
   \s*4 .* (re)
-  $ rm -f "$TMP/filtered_strains.txt"
+
+By setting the subsample seed above, we should guarantee that we get the same "random" strains as another run with the same command.
+
+  $ ${AUGUR} filter \
+  >  --metadata filter/metadata.tsv \
+  >  --group-by region \
+  >  --sequences-per-group 1 \
+  >  --subsample-seed 314159 \
+  >  --output-strains "$TMP/filtered_strains_repeated.txt" > /dev/null
+
+  $ diff -u <(sort -k 1,1 "$TMP/filtered_strains.txt") <(sort -k 1,1 "$TMP/filtered_strains_repeated.txt")
```

Comment:
Why is `-k 1,1` needed here?

Reply:
It's not. Old habits from working with BED files die hard. I just removed it.

Reply:
:joy: I was trying to figure out what I was missing…